### ISSUE TITLE
Questions about the pipeline

### ISSUE THREAD

**Issue by dulucas** [balakg/posewarp-cvpr2018#2]
Hi Guha,

     Glad that you've shared your work, and I've tested on the Human 3.6M dataset, the results are magnificent!
     And here are some questions:
     1. For the loss function, why you choose the VGG instead of Resnet? Is there any benefits? (As I know, in the area of single image depth prediction, the depth images generated by VGG are more sharper of Resnet's)
     2. Have you tried on any difficult poses? the poses from extreme sports for exemple. 
      Thanks!

**Comment by balakg** [comment]
Great to hear!

1. Resnet may also work fine. I chose VGG because it was used in some past work for texture synthesis (https://arxiv.org/pdf/1505.07376.pdf) and view synthesis (https://arxiv.org/pdf/1703.02921.pdf). The different blocks of VGG have also shown to correspond to progressively more complex image features, so you can choose blocks depending on what to emphasize. But presumably, resnet trained for image classification would also work.

2. The poses we tried were limited to fitness, golf and tennis videos, so no, we haven't tried it on more extreme sports yet. Though I think some of the poses in our videos can be quite complex.

**Comment by dulucas** [comment]
Thanks a lot !
What if there are some key points are missing?

**Comment by babyjia** [comment]
Will,I would like to know that where is the training dataset?

**Comment by dulucas** [comment]
http://vision.imar.ro/human3.6m/description.php
Here is the H3.6M dataset

**Comment by babyjia** [comment]
Ｉ am sorry to bother you that I coun't find the test code

**Comment by balakg** [comment]
@lucasdu007 You mean if there are joints missing?

**Comment by dulucas** [comment]
@balakg yes, if there are joints missing, what should I do?

**Comment by balakg** [comment]
If there are joints missing, those heat maps remain blank.
